spec/spec_helper.rb

```diff
@@ -12,7 +12,12 @@
 require 'support/factories'
 require 'support/query_tracker'
 
-ActiveSupport::Deprecation.behavior = :raise
+if ActiveSupport.respond_to?(:deprecator)
+  ActiveSupport.deprecator.behavior = :raise
+  ActiveRecord.deprecator.behavior = :raise
```

Comment:
I _think_ this is right, but would appreciate another pair of eyes to check

Reply:
I'm not super familiar with the changes to deprecations, but this looks good to me. It [looks like ](https://github.com/rails/rails/blob/404ce7e8dec6de842a985b7efee6615f4e3af0da/activerecord/lib/active_record/deprecator.rb)`ActiveRecord.deprecator` is a dedicated instance of `ActiveSupport::Deprecation`.